pkg/edgeview/vendor/modules.txt

```diff
@@ -91,6 +91,16 @@ github.com/vishvananda/netns
 # github.com/yusufpapurcu/wmi v1.2.2
 ## explicit; go 1.16
 github.com/yusufpapurcu/wmi
+# golang.org/x/crypto v0.14.0
+## explicit; go 1.17
```

Comment:
It doesn't work with go version 1.21?

Reply:
well, even i checkout the latest of golang.org/x/crypto, which is v1.22.0, the go is v1.18 after the 'go mod vendor'